### PR TITLE
 bug fixes and styling (cli)

### PR DESCRIPTION
**Pull request by scientific-dev**
## Things Changed

- Fixed imports in the [58b674c](https://github.com/DevSnowflake/create-discord-app/commit/58b674c8891edc92467d84701fac612472aed06b) commit.
- Styled code and removed useless code.
- Added `--gitCommit` and `--noGit` as subcommands.
- Added new aliases.

### CODE REVIEW

**Comment by scientific-dev** [comment]
> just use `yargs` instead. [Check](https://github.com/yargs/yargs/blob/master/docs/advanced.md#example-command-hierarchy-using-indexmjs)

Fixed.

**Comment by rhygg** [comment]
@scientific-dev should I merge this? If you've got everything pushed?

File: bin/commands.js [hunk 1]
```diff
@@ -19,10 +19,14 @@ const libraries = {
...
-    ${chalk.whiteBright("--discord")}       : Shows discord support server invite
-    ${chalk.whiteBright("--help")}          : Shows help menu
-    ${chalk.whiteBright("--version")}       : Shows CDA version
-    ${chalk.whiteBright("--create")}        : Create Project
+    ${chalk.whiteBright("--discord")}       : Shows discord support server invite.
+    ${chalk.whiteBright("--help")}          : Shows help menu.
+    ${chalk.whiteBright("--version")}       : Shows CDA version.
+    ${chalk.whiteBright("--create")}        : Create Project.
+
+    Subcommands:
```
Comment (rhygg) [inline bin/commands.js:27]: could you add `--dir` to the sub-commands?

**Review by rhygg — CHANGES_REQUESTED** [review]
I think, other than that everything's alright.

File: bin/commands.js (hunk 1 above)
Comment (scientific-dev) [reply bin/commands.js:27]: Yeah added `--force` sub command too.

**Comment by scientific-dev** [comment]
> @scientific-dev should I merge this? If you've got everything pushed?

Yes its ready to be merged.

**Review by rhygg — APPROVED** [review]
Awesome time to merge!

**Pull request merged**